manifests/params.pp

```diff
@@ -4,14 +4,49 @@
 # It sets variables according to platform.
 #
 class reaktor::params {
+
+  $ensure       = 'present'
+
+  $manage_user  = true
+  $user         = 'reaktor'
+  $homedir      = '/opt/reaktor'
+  $shell        = '/usr/sbin/nologin'
+  $uid          = 4500
+
+  $manage_group = true
+  $group        = 'reaktor'
+  $gid          = 4500
+
+  $repository   = 'https://github.com/pzim/reaktor.git'
+
+  # reaktor configs
+  $address              = $::fqdn
+  $port                 = 4570
+  $servers              = 1
+  $max_conns            = 1024
+  $max_persistent_conns = 512
+  $timeout              = 30
+  $environment          = 'production'
+  $pid                  = 'tmp/pids/reaktor.pid'
+  $log                  = 'reaktor.log'
+
+  # masters.txt file
+  $manage_masters = true
```

Comment:
the default from line 10 to 34 could be moved to init.pp as they do not depend a condition